testsuite/MDAnalysisTests/data/Amber/multi_ache.prmtop

```diff
@@ -0,0 +1,4506 @@
+%VERSION  VERSION_STAMP = V0001.000  DATE = 07/20/23  21:22:41
```

Comment:
304 kb for a new optional case is quite a lot. Can we bz2 this file please?

Reply:
Done.